test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorPageTest.cs

```diff
@@ -286,9 +286,9 @@ public static TheoryData<HtmlEncoder> HtmlEncoderData
             // Arrange
             var page = CreatePage(v =>
             {
-                v.Path = "/Views/TestPath/Test.cshtml";
+                v.Path = "/Views/Shared/_Layout.cshtml";
                 v.RenderSection("bar");
-            });
+            }, context: CreateViewContext(viewPath: "/Views/TestPath/Test.cshtml"));
```

Comment:
Do we still want a new test now that we're going with the end view? Because it seems like this is the exact scenario I'm working on, I don't really know what other test I would make.